File: VS Code/C++/Recurssion/power of a number.cpp
#include<iostream>
using namespace std;
int power(int p,int q){
	if(q==0) return 1;
	return p*power(p,q-1);
}
int main(){
	int p,q;
	cout<<"Enter base number : ";
	cin>>p;
	cout<<"Enter power of that number : ";
	cin>>q;
	
	int result=power(p,q);
	cout<<result;
	return 0;
}

File: VS Code/Python/recursion/power of a number.py
def power(x,y):
    if y==0:
        return 1
    elif y==1:
        return 2
    else:
        return (x*power(x,y-1))

num1 = int(input("Enter the base: "))
num2 = int(input("Enter the power: "))
print(power(num1,num2))